app/models/spina/navigation_item.rb

```diff
@@ -25,6 +25,14 @@ def menu_title
       end
     end
 
+    def materialized_path
```

Comment:
This could be rewritten as:
```ruby
def materialized_path
  page&.materialized_path || url
end
```

or like this if you prefer something a bit more explicit:

```ruby
def materialized_path
  return page.materialized_path if page
  url
end
```

Reply:
Great suggestion. I went ahead and made those corrections.